bug(macros): #[routes] generates WsUrls::ws() accessor outside __namespaced_ws_resolvers — E0451 private field

## Bug Description

The `#[routes]` macro generates `impl ResolvedUrls { pub fn ws() -> WsUrls }` **outside** `__namespaced_ws_resolvers` where `WsUrls` is defined. Because `WsUrls.resolver` is private, struct-literal `WsUrls { resolver: self }` in the outer impl block fails with E0451.

By contrast, the HTTP-side `impl ResolvedUrls { pub fn server() -> ServerUrls }` is emitted **inside** `__namespaced_resolvers` where `ServerUrls` is defined — it compiles. The WebSocket equivalent was placed outside its module.

## Steps to Reproduce

1. Create a Reinhardt project with `installed_apps!` registering at least one app.
2. Create `ws_urls.rs` with `#[url_patterns(InstalledApp::auth, mode = ws)]`.
3. Declare `pub mod ws_urls;` in the app entry module.
4. Use `#[routes]` on the router function.
5. `cargo check`

```
error[E0451]: field `resolver` of struct `__namespaced_ws_resolvers::WsUrls` is private
  = note: this error originates in the attribute macro `routes`
```

## Expected Behavior

`cargo check` passes. `urls.ws().auth()` accessible same as `urls.server().auth()`.

## Root Cause

In `crates/reinhardt-core/macros/src/routes_registration.rs`, the `impl ResolvedUrls { pub fn ws() }` block is emitted **outside** `mod __namespaced_ws_resolvers`, while `WsUrls` (with private `resolver` field) is defined inside. Fix: move this impl block inside `__namespaced_ws_resolvers` (parallel to the HTTP `server()` pattern).

## Environment

- Rust version: 1.94.0-nightly
- reinhardt-web revision: `f318c8d6` (main, after PR #3830)
- OS: macOS 25.3

## Related

- Introduced by #3782 (`#[url_patterns(mode = ws)]`)
- Also relevant: #3830 (`WebSocketRouter::with_namespace`)

--- a/crates/reinhardt-core/macros/src/routes_registration.rs
+++ b/crates/reinhardt-core/macros/src/routes_registration.rs
@@ -820,6 +820,15 @@ pub(crate) fn routes_impl(args: TokenStream, input: ItemFn) -> Result<TokenStrea
 					impl WsUrls<'_> {
 						#(#ws_app_accessors)*
 					}
+
+					// urls.ws() accessor lives inside this module so that WsUrls.resolver
+					// (a private field) is accessible via struct-literal syntax (E0451 fix).
+					impl ResolvedUrls {
+						/// Access WebSocket URL resolvers via `urls.ws().<app>().<route>()`.
+						pub fn ws(&self) -> WsUrls<'_> {
+							WsUrls { resolver: self }
+						}
+					}
 				}
 				#[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
 				pub use __namespaced_ws_resolvers::*;
@@ -834,15 +843,6 @@ pub(crate) fn routes_impl(args: TokenStream, input: ItemFn) -> Result<TokenStrea
 							"WebSocket URL resolution requires reinhardt-websockets. \
 							 Call impl_ws_url_resolver!(ResolvedUrls) to enable it."
 						)
-					}
-				}
-
-				// Add urls.ws() accessor to ResolvedUrls
-				#[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
-				impl ResolvedUrls {
-					/// Access WebSocket URL resolvers via `urls.ws().<app>().<route>()`.
-					pub fn ws(&self) -> WsUrls<'_> {
-						WsUrls { resolver: self }
 					}
 				}
 

